Lux/lux_react/src/components/JenSwitch.js

```diff
@@ -5,7 +5,7 @@ import Checkbox from "@mui/material/Checkbox";
 import FormControlLabel from "@mui/material/FormControlLabel";
 
 function JenSwitch({ json, onChange }) {
-    const [switchValue, setSwitchValue] = useState(json.default_value || false);
+    const [switchValue, setSwitchValue] = useState(json.value || false);
```

Comment:
consider nullish coalescing here ``` ?? false ```

Reply:
Yes thanks this is correct - using ??  in JenMenu.js fixes a bug that occurred when default menu choice was zero.